src/jquery.mockjax.js

```diff
@@ -668,7 +678,7 @@
 
 	var DEFAULT_RESPONSE_TIME = 500;
 
-    
+
 	$.mockjaxSettings = {
```

Comment:
We need to add a default in here, and I think it should probably be `null` (versus the empty string which you have in the readme).